edb/lib/std/25-enumoperators.edgeql

```diff
@@ -105,3 +105,9 @@ CREATE CAST FROM std::anyenum TO std::str {
     SET volatility := 'IMMUTABLE';
     USING SQL CAST;
 };
+
+# The reverse cast from json to enum is handled in compile_cast
```

Comment:
I think we still want an explicit declaration of the reverse cast. Use `USING SQL EXPRESSION` to mark it as "compiler-handled".